assets/src/components/form/select/model-selects/ticket.js

```diff
@@ -105,6 +105,7 @@ export default class TicketSelect extends Component {
 			...this.props,
 			...selectOpts,
 			...this.state,
+			optionsEntityMap,
```

Comment:
This shouldn't be needed because its provided by `...this.props` (via `defaultProps`).